test/logger_test.cc

```diff
@@ -2,6 +2,7 @@
 #include "gtest/gtest.h"
 
 TEST(TestLogger, DummyTest) {
-  BOOST_LOG_TRIVIAL(trace) << "A trace severity message";
+  blsb::LoggerType local_logger;
```

Comment:
@malirod for me it looks like overhead when one has to create logger variable manually for every function